athena-jdbc/README.md

```diff
@@ -178,13 +178,19 @@ A partition is represented by two partition columns of type varchar. We leverage
 
 **Note:** In case of Redshift partition_schema and partition_name will always be '*'. It does not support external partitions. Performance with huge datasets is slow.
 
+### Running Integration Tests
+
+See [Running Integration Tests](https://github.com/awslabs/aws-athena-query-federation/blob/master/athena-federation-integ-test/README.md#running-integration-tests).
+
```

Comment:
in addition to the link, can you maybe put the commands here? i thought its just the build of the integ test and then the run of the tests.

Reply:
Also, what settings do i set that are connector specific in that test config?

Reply:
Added additional info.